LiveSplit.AutoSplitters.xml

```diff
@@ -13821,3 +13821,15 @@
         <Website>https://github.com/Marek2810/Vlak</Website>
     </AutoSplitter>
 </AutoSplitters>
```

Comment:
@SabulineHorizon You need to place your <AutoSplitter></AutoSplitter> section before the closing </AutoSplitters> Tag.

Reply:
I have now edited it. I think the tag should be in the correct position now.

Reply:
I just now realized that you made the change and that I could have simply confirmed it. Should I go ahead and confirm your edit and cancel my last one, or leave it as is? I'm new to GitHub, so I don't know what the standard procedures are yet.

Reply:
Nah, I didn't change the code. It was only asking you to accept my review or not 😅